src/adr/0005-distributed-task-termination-ackowledgement.md

```diff
@@ -0,0 +1,24 @@
+# 5. Distributed Task termination ackowledgement
+
+Date: 2019-10-02
+
+## Status
+
+Accepted (lazy consensus)
+
+## Context
+
+By switching the task manager to a distributed implementation, we need to be able to execute a `Task` on any node of the cluster.
+We need to have a way to broadcast some `Event`s (`CancelRequested` and termination `Event`s).
+
+## Decision
+
+ * Creating a `RabbitMQEventHandler` which publish `Event`s to RabbitMQ
+ * the `Completed`, `Failed`, `Canceled` events will be broadcasted on an exchange
```

Comment:
could we have a functional description of that?

Reply:
What is a 'functional description'?

Reply:
a non-technical business rule description? (I may lack the good words in english)